manifests/jetty.pp

```diff
@@ -110,6 +106,13 @@
     mode    => '0644',
     recurse => false,
     notify  => Class['shibboleth_idp::service'],
+  } ->
+
+  file { $idp_jetty_log_dir:
+    ensure => directory,
+    owner  => $shibboleth_idp::shib_user,
+    group  => $shibboleth_idp::shib_group,
+    mode   => '0644',
```

Comment:
Jetty base was recursive, but the individual directories being created were not (recurse was present but set to false). I've cleaned all that up and added the log directory variable to the list since the separate resource block was doing basically the same work.

I'm not sure how the jetty_base and jetty_files array will interact though. Most of the directories being created by jetty_files are under jetty_base. Will the recurse on jetty_base apply to them after the initial creation or will puppet not recurse through them since their file resource does not set recurse = true?